src/unix/apple/macos/component/arm.rs

```diff
@@ -50,7 +53,15 @@ impl ComponentsInner {
         &mut self.components
     }
 
+    #[allow(unreachable_code)]
     pub(crate) fn refresh_list(&mut self) {
+        // See issue https://github.com/GuillaumeGomez/sysinfo/issues/1279
+        if self.has_been_called {
```

Comment:
That's what I meant in previous comment: on mac, this function should do nothing.

Reply:
I now changed the function to immediately return and also changed the comment to reflect this change.